lib/MusicBrainz/Server/Data/Edit.pm

```diff
@@ -625,7 +625,7 @@ sub load_all
         while (my ($model, $ids) = each %$edit_references) {
             $objects_to_load->{$model} ||= [];
             if (ref($ids) eq 'ARRAY') {
-                $ids = [ uniq grep { defined } @$ids ];
```

Comment:
This change can't hurt, but can we also figure out why some return undef and some ''?

Reply:
Well, foreign_keys come from the edit data, so they're stored in the edit data like that.